copier/main.py

```diff
@@ -44,6 +44,32 @@
     from backports.cached_property import cached_property
 
 
+class CopierAnswersInterrupt(Exception):
```

Comment:
Please move it to errors.py

Reply:
Ack. May lose the type hinting to avoid circular imports (user_data imports errors, errors would need to import user_data).

Reply:
You can use this:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:  # always false
    from copier... import user_data
```

Reply:
Ack, done. 

Note to folks that have hibernated on modern python roughly boiled down to this:

```
from typing import TYPE_CHECKING

if TYPE_CHECKING:  # always false
    from .user_data import AnswersMap, Question, Template

...
# In the constructor, use string references to the type names instead of the imported Objects.
class CopierAnswersInterrupt(CopierError, KeyboardInterrupt):
    def __init__(self, answers: 'AnswersMap', last_question: 'Question', template: 'Template') -> None:
```

The stringly references could be removed in 3.7+ as per this SO comment: https://stackoverflow.com/a/39757388 

I couldn't find any references to `from __future__ import annotations` to backport that functionality to 3.6 (listed as one of the pyproject.toml classifiers currently). Not worth saving a couple of quotes IMO.